common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeAuth.java

```diff
@@ -13,4 +13,15 @@ public BridgeAuth(String tmp) {
     public String getToken() {
         return token;
     }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("BridgeAuth{");
+        sb.append("token='");
+        sb.append(token.substring(0, 10));
+        sb.append("...").append(token.substring(token.length() - 10))
+                        .append('\'');
```

Comment:
How long is the token? Does this code hide enough chars to avoid leaking it? I'm sure the answer is yes but I still have to ask 😃

Reply:
Something like 2k.